modules/router/templates/assets_origin.conf.erb

```diff
@@ -61,6 +61,10 @@ server {
     proxy_pass <%= @upstream_ssl ? 'https' : 'http' %>://static.<%= @app_domain %>;
   }
 
+  location /media/ {
+    proxy_pass <%= @upstream_ssl ? 'https' : 'http' %>://static.<%= @app_domain %>;
+  }
+
```

Comment:
I think it would be worth mentioning in the commit note that previously we were relying on the implicit behaviour in the final `location` block for paths matching `/`.

Reply:
I've added the following to the commit note:

> Note that requests for /media/ would've previously been handled by the catch-all route (`location /`) in this file.